docs/contributing/code-style.md

```diff
@@ -28,7 +28,7 @@ Look at the surrounding code, or a similar part of the project, and try
 to do the same thing. If you think the other code has actively bad
 style, fix it (in a separate commit).
 
-When in doubt, ask in [chat.zulip.org](https://chat.zulip.org).
+When in doubt, ask in [The Zulip developer community](https://zulip.com/developer-community/#narrow/stream/49-development-help).
```

Comment:
The `The` should not be capitalized anywhere unless it is the first word in the sentence.

Reply:
> The `The` should not be capitalized anywhere unless it is the first word in the sentence.

Okay!